ports/stm/boards/arduino_uno_q/mpconfigboard.mk

```diff
@@ -0,0 +1,36 @@
+USB_VID = 0x0483
```

Comment:
Does assigning a VID and PID make sense here? By my reading of the UNO Q datasheet (see page 13: 4.2 Block Diagram), it doesn't look like the STM32U585 has access to the USB-C port. It seems the board is set up for Debian running on the Dragonwing MPU to program the STM32 over SWD and do RPC bridge stuff over UART and SPI channels.